flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSClientUtil.java

```diff
@@ -207,6 +207,39 @@ S createAwsSyncClient(
                 .build();
     }
 
+    public static <
```

Comment:
can we collapse the previous `createAwsSyncClient()` into this one?

Reply:
sure